src/server/db/schema.ts

```diff
@@ -2,7 +2,9 @@ import { InferSelectModel, relations } from 'drizzle-orm';
 import { index, int, integer, primaryKey, sqliteTable, text } from 'drizzle-orm/sqlite-core';
 import { type AdapterAccount } from 'next-auth/adapters';
 
-export const users = sqliteTable('user', {
+// workaround for typescript check in adapter
+// preferably add email into credential login and make email non-nullable here
```

Comment:
Can you elaborate why this workaround is needed?

Reply:
If this is a necessity, force the email in the existing credential system. IMO.
At first it wasn't even supposed to be in, to make the login update as simple as possible, but ended up adding it because we thought it would be useful in the future.
If it's a necessity, then it's fine to make it so, we just didn't have a reason until now since it's all new.